Lib/typing.py

```diff
@@ -428,8 +428,15 @@ def __getitem__(self, parameters):
         return self._getitem(self, *parameters)
 
 
-@_SpecialForm
-def Any(self, parameters):
+class _AnyMeta(type):
+    def __instancecheck__(self, obj):
+        raise TypeError("typing.Any cannot be used with isinstance()")
+
+    def __repr__(self):
```

Comment:
This needs `__or__`/`__ror__` so we can use it in unions.

Reply:
Does it? `Any` is just a normal class. This is tested in `test_types` as well

Reply:
Yeah. As long as it's a normal Python class/type, the metaclass's ``__or__`` is automatically a `types.UnionType` operation.